frontend/src/photoView/photoView.scss

```diff
@@ -1,3 +1,23 @@
 .image-photo {
     width: 100%;
 }
+line {
```

Comment:
d'oh, we don't have a css linter for this project. spaces after colons and consistent indentation, plz.